### ISSUE TITLE
Bug with upper case variables

### ISSUE THREAD

**Issue by luispfonseca** [luispfonseca/stata-rcallcountrycode#2]
``` stata
. rcallcountrycode A, from(country.name.en) to(country.name.en) gen(std_name) marker
Preparing data to send to R
Calling R...
[1] "Using countrycode package version: 1.1.0"
A not found
```

**Comment by luispfonseca** [comment]
`import delimited` in stata makes variables lower case when importing

**Comment by luispfonseca** [comment]
fixed in dc7dfc0
